Orange/widgets/unsupervised/owmanifoldlearning.py

```diff
@@ -260,6 +262,8 @@ def apply(self):
                             self.Error.n_neighbors_too_small("{}".format(n))
                         else:
                             self.Error.manifold_error(e.args[0])
+                    except np.linalg.linalg.LinAlgError as e:
+                        self.Error.manifold_error(e.args[0])
```

Comment:
Use `str(e)` instead of `e.args[0]`.
 Calling `str()` on an instance of `BaseException` will give [something at least as good as `e.args[0]](https://docs.python.org/3/library/exceptions.html#BaseException). 

Actually, even `self.Error.manifold_error(e)` but I'd prefer being explicit.